libnmstate/ifaces/ifaces.py

```diff
@@ -44,6 +44,7 @@
 from .vlan import VlanIface
 from .vxlan import VxlanIface
 from .vrf import VrfIface
+from .wireguard import WireguardIface
```

Comment:
the WireGuard maintainers have a strong opinion about the spelling and styling of WireGuard and the API.

In particular, they wish that `WireGuard` is spelled with upper `G`. NetworkManager's API does that too, at places where there is a CamelCase identifier (but it violates that are places like `NM_SETTING_WIREGUARD_SETTING_NAME` or `nm_wireguard_peer_new()`. By doing that, NetworkManager already violates it's own style, because from `NMSettingWireGuard` one would normally expect `NM_SETTING_WIRE_GUARD_SETTING_NAME`.

They probably would also like to see that the configuration options are spelled like why are in `man wg`. That is, `AllowedIPs` and `PresharedKey` instead of `allowed-ips` and `preshared-key`. However, that style already conflicted with NetworkManager's API, so NetworkManager does not do that either. So never mind that.

It's only about `WireGuard` instead of `Wireguard`.

Reply:
Hi @thom311, I will make sure to correct it in my next commit. Thanks a lot for the review. :)

Reply:
@cathay4t What do you think? Should we break our existing style? I do not like the idea.. but it's true that they have strong opinions on that.

Reply:
For internal stuff like class name, you can follow the guidance of WireGuard. But for public property, please use lower case with dash for consistency